daily_post.py

```diff
@@ -17,10 +19,12 @@
                     user_agent=USER_AGENT,
                     username=USERNAME)
 
-subreddit = reddit.subreddit('SeattleWA')
+subreddit = reddit.subreddit(SUB)
 
 now = datetime.datetime.now()
 
+WEEKDAY = calendar.day_name[now.weekday()]
```

Comment:
This will break your template since this returns the named day of week (e.g. `Wednesday`).Iinstead let's just assign it `now.weekday() and update Ln53 to use this variable (And also drop the import on calendar). After that, should be good!